src/test/ui/const-generics/defaults/complex-unord-param.rs

```diff
@@ -0,0 +1,28 @@
+// run-pass
+// Checks a complicated usage of unordered params
+
+#![feature(const_generics)]
+#![allow(incomplete_features)]
+#![allow(dead_code)]
+
+struct FixedOutput<'a, const N: usize, T=u32> {
+  out: &'a [T; N],
+}
+
+trait FixedOutputter {
+  fn out(&self) -> FixedOutput<'_, 10>;
+}
```

Comment:
I don't think `FixedOutput` is used in the rest of the example...

It might make sense to move it into a different test, maybe `simple-defaults.rs` or sth :thinking: